src/hydra/hydraClient.js

```diff
@@ -302,7 +302,7 @@ export default ({entrypoint, resources = []}, httpClient = fetchHydra) => {
           .then(data => ({data, total: response.json['hydra:totalItems']}));
 
       case DELETE:
-        return Promise.resolve(() => ({data: {}}));
+        return Promise.resolve(response => (response.data));
```

Comment:
parenthesis not needed in the `(response.data)`

Reply:
Or you can use `destructuring`:

```js
({ data }) => data
```

Reply:
Just to notice: in DELETE_MANY you just do basically:

```return {data: {}};```

which fixes also this error :smile:

Reply:
You can delete without using delete_many

Reply:
@mauchede using destructuring generate more codes with yarn build than just this.

Reply:
@Simperfit is no js-hipster :cry: